src/app/legacy/psammead/psammead-heading-index/src/index.jsx

```diff
@@ -11,11 +9,6 @@ const HeadingIndex = styled.h1`
   margin: 0;
 `;
 
-HeadingIndex.propTypes = {
-  script: shape(scriptPropType).isRequired,
-  service: string.isRequired,
-};
-
 HeadingIndex.defaultProps = {
```

Comment:
this has been missed

Reply:
These are Emotion styled components, so I'm not sure of the best way to set ES6 default values on these. React doesn't seem to mind these even with version `18.3.x`